relation/core.md

```diff
@@ -22,3 +22,29 @@ The standard error is 3 standard deviation of the mean of the sample.
 ### relation:StandardDeviation {#StandardDeviation}
  
  Dispersion of the values of a random variable around its expected value.
+
+### relation:Confidence {#MarginOfError}
+ 
+Use `Confidence` to specify the level of uncertainty in the estimate. It can be used multiple times,
```

Comment:
> Use `Confidence` to specify ~~the level of~~ uncertainty in the estimate. 


If i were to read something about a level of uncertainty, I would immediately think about some confidence-level (i.e. 99% or 95%)